circle.yml

```diff
@@ -26,4 +26,4 @@ dependencies:
 test:
   override:
     - apm test
-    - coala --ci
```

Comment:
I was under the assumption that deprecated binaries should be updated to avoid future problems and backports, as was the case with `format_str`.